SelectedPlace.js

```diff
@@ -0,0 +1,222 @@
+								{this.parseDate(this.props.weatherDetails.dt)}
+							</Text>
+
+							<Text style={[styles.text_style, styles.headerTemp]}>
+								MIN: {this.props.weatherDetails.main.temp_min }{this.props.unit}, 
+								MAX: {this.props.weatherDetails.main.temp_max }{this.props.unit}
+							</Text> 
+						</View>
+						<View style={styles.header_right_col}>
+							<Picker
+								selectedValue={this.props.unitType}
+								style={{height: 30, width: 150}}
+								onValueChange={
+									(itemValue) => this.props.updateUnitType(this.props.weatherDetails.name,itemValue)
+								}
+							>
+								<Picker.Item label="Celsius " value="metric" />
+								<Picker.Item label="Kelvin " value="default" />
+								<Picker.Item label="Fahrenheit " value="imperial" />
+							</Picker> 
+						</View>
+
+					</View>
+
+					<View style={styles.mid_container}>
+						<Image
+						  style={styles.weather_icon}
+						  source={{uri: this.getIcon(this.props.weatherDetails.weather[0].icon)}}
+						/>
+						
+						<Text style={[styles.text_style, styles.midTemp]}>
+							{this.props.weatherDetails.main.temp}{this.props.unit_temp}
+						</Text>
+
+						<Text style={styles.text_style}>
+							{this.props.weatherDetails.weather[0].description}
+						</Text> 
+					</View>
+
+					<ForecastWeather forecastDetails={this.props.forecastDetails} unit_temp={this.props.unit_temp}/>
+
+					<View style={styles.addDetails_container}>
+						<View style={styles.text_row}>
+							<Text style={[styles.left_col_style, styles.text_row_style, styles.headerDate]}>
+								Humidity	
+							</Text>
+							<Text style={[styles.text_row_style, styles.headerDate]}>
+								: {this.props.weatherDetails.main.humidity} %
+							</Text>
+						</View>
+
+						<View style={styles.text_row}>
+							<Text style={[styles.left_col_style, styles.text_row_style, styles.headerDate]}>
+								Wind	
+							</Text>
+							<Text style={[styles.text_row_style, styles.headerDate]}>
+								: {this.props.weatherDetails.wind.speed} {this.props.unit_speed}
+							</Text>
+						</View>
+
+						<View style={styles.text_row}>
+							<Text style={[styles.left_col_style, styles.text_row_style, styles.headerDate]}>
+								Pressure	
+							</Text>
+							<Text style={[styles.text_row_style, styles.headerDate]}>
+								: {this.props.weatherDetails.main.pressure} hPa
+							</Text>
+						</View>
+					</View>
+				</View>
+			);
+		}
+	}
+}
+
+const styles = StyleSheet.create({
+	content_container_selected:{
+		flex: 1,
+		paddingTop: 65,
+		padding: 10,
+	},
+	
+	header_container:{
+		flex: 1,
+		flexDirection: 'row',
+		// backgroundColor: 'red'
+	},
+	mid_container:{
+		flex: 3,
+		// backgroundColor: '#a3c6cd',
+		alignItems: 'center',
+		justifyContent: 'center'
+	},
+	forecast_container:{
+		flex: 2,
+		// backgroundColor: '#fff',
+
+	},
+	addDetails_container:{
+		flex: 1,
+		alignItems: 'center',
+		justifyContent: 'center',
+		// backgroundColor: 'yellow'
```

Comment:
We expect a production ready code. You need to clear every unused code not a comment.